app/Models/User.php

```diff
@@ -204,6 +204,18 @@ public function scopeWithLastName(Builder $query, $lastname)
         return $query->where(DB::raw('LOWER(lastname)'), 'like', '%'.strtolower($lastname).'%');
     }
 
+    /**
+     * Scope a query to only get users that have a email like the passed one.
```

Comment:
Proposal: "Enter a query to only get users who have an email like the passed one."